Source/WebCore/Modules/applicationmanifest/ApplicationManifest.h

```diff
@@ -57,12 +57,15 @@ struct ApplicationManifest {
         OptionSet<Purpose> purposes;
     };
 
+    String rawJSON;
     String name;
     String shortName;
     String description;
     URL scope;
+    bool isDefaultScope;
```

Comment:
Please add a default initializer to avoid uninitialized memory. E.g.:
```
bool isDefaultScope { true };
```
or more likely this given your code below:
```
bool isDefaultScope { false };
```